scripts/delete-unverified-subscribers.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const DB = require("../db/DB");
+
+
+async function deleteUnverifiedSubscribers() {
+  await DB.deleteUnverifiedSubscribers();
+}
+
+
+(async () => {
+  await deleteUnverifiedSubscribers();
```

Comment:
Why not just call the `DB.deleteUnverifiedSubscribers()` directly instead of the proxy func?